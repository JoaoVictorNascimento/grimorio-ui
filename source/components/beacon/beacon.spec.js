import '../../../internals/test/helper';

import Beacon from './beacon-component';

/** @test {Breadcrumb} */
describe('Beacon component', () => {
/** @test {Breadcrumb#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Beacon />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
