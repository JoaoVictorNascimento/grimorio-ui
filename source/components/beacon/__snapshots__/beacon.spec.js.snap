// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Beacon component #render render correctly 1`] = `"<span className=\\"minhaClasse beacon\\" />"`;
